.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "standard"
+}
```

Comment:
Not important, but please turn on"new line at end of file" settings in your IDE. It will help out my OCD haha

Reply:
It didn't pick it up due to being a hidden file I think; I will add a newline in all PRs